### DISCUSSION TITLE
Failed to Download Disney Plus video

### DISCUSSION THREAD

**Discussion by jummama1** [aclap-dev/video-downloadhelper#696] [Help and Q&A]
### Your question

It started downloading the video and right before it is down, it says error downloading video, using google chrome. 

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.disneyplus.com/play/42d666c2-85e7-4810-b5a7-4c3622ed4bfb",
  "error": {
    "details": "Error: Native port disconnected",
    "downloadable_id": "downloadable_406363641455803",
    "error": "coapp_failure",
    "id": "coapp_failure_17803d09-fb03-4905-b2f5-bdee13ad087b"
  }
}

**Comment by jcfrog** [comment]
I guess Disney plus is using DRM.
